examples/vulkan/vulkan.h

```diff
@@ -42,80 +42,81 @@
 #define GLAD_OPTION_VULKAN_HEADER_ONLY
 
 #ifdef __cplusplus
-extern "C" {
+extern "C"
+{
```

Comment:
File should be excluded as well

Reply:
Good catch. I found that stb_perlin.h was getting formatted as well so I excluded that file and reverted the formatting.